src/sales/order-ship.md

```diff
@@ -53,3 +55,31 @@ An order that has been paid, but is awaiting shipment has the `Processing` statu
    - To post the comment in the customer’s account, select the **Visible on Frontend** checkbox.
 
 1. Click <span class="btn">Submit Comment</span>.
+
+## To track the shipment:
+
+**Method 1:** From order information tab
+
+1. On the Admin menu, select **Sales** > **Orders**.
+
+1. Find the shipment order in the grid, and click **View**.
+
+1. Scroll down to the **Shipping & Handling Information** section.
+
+1. Click **Track Order**.
```

Comment:
This can be streamlined into a single step. We try to reserve bold for those elements that are "touched" by the user.

```suggestion
1. Scroll down to the _Shipping & Handling Information_ section and click **Track Order**.
```